rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -143,6 +144,100 @@ impl PrometheusVm {
         self
     }
 
+    /// Expects the layout of directory to be like:
+    /// ```
+    /// root
+    /// ├── folder1
+    /// │   ├── kustomization.yaml
+    /// │   ├── dashboard1.json
+    /// │   └── dashboard2.json
+    /// ├── folder2
+    /// │   ├── dashboard3.json
+    /// │   ├── dashboard4.json
+    /// │   ├── kustomization.yaml
+    /// ...
+    /// ```
+    ///
+    /// This process automatically discovers all `*.json` files, which are interpreted as Grafana dashboards. It then copies these files to a destination, where they will be sent to the Prometheus VM for use with the testnets. The expected name of the dashboards directory is determined by reading the `commonAnnotations.k8s-sidecar-target-directory` path from the `kustomize.yaml` file. This value specifies the location where the dashboards should be placed so that the links don't get broken.
+    fn transform_dashboards_root_dir(logger: Logger) -> Result<()> {
+        let destination = get_dependency_path("rs/tests/dashboards");
```

Comment:
@basvandijk I'm not super familiar with this. Is it ok to write to this directory? See `create_dir_all` below.

Reply:
No, paths returned by `get_dependency_path(...)` are meant to be read-only dependencies. 

If you need to write please write to [`env.get_path(GRAFANA_DASHBOARDS)`](https://sourcegraph.com/github.com/dfinity/ic@659e51016cab1b2f82466941a9077b35278f3a76/-/blob/rs/tests/driver/src/driver/test_env.rs?L92:12-92:20) where you define: 
```rust
const GRAFANA_DASHBOARDS: &str = "grafana_dashboards";
```
You can get the `env` from the `start(&self, env: &TestEnv)` function below and pass it to `transform_dashboards_root_dir`.